Dockerfile.test-tools

```diff
@@ -0,0 +1,24 @@
+# based on build-tools but adds postgresql-client and postgresql packages
+ARG BUILD_TOOLS_BASE_IMAGE
+
+FROM ${BUILD_TOOLS_BASE_IMAGE}
+
+# Switch to root user
+USER root
+
+# postgresql (depends on  curl)
+# needed in some tests because neon artifacts cannot be used 
+# in combination with staticaclly built openssl
+# due to https://github.com/neondatabase/neon/issues/8275
+ENV PG_VERSION=16
```

Comment:
Let's use `ARG` instead of `ENV`. 
Environment variables that we set in the Dockerfile also exist in the container. I'm afraid `PG_VERSION` can be accidentally picked up by some of our scripts/tools

Reply:
makes sense